pkgdb/src/search/params.cc

```diff
@@ -57,7 +57,24 @@ SearchQuery::check() const
   if ( this->partialMatch.has_value() && this->partialNameMatch.has_value() )
     {
       throw ParseSearchQueryException(
-        "`partialmatch' and `partialNameMatch' filters "
+        "`partialMatch' and `partialNameMatch' filters "
+        "may not be used together." );
+    }
+  /* `partialMatch' and `partialNameOrRelPathMatch' cannot be used together. */
+  if ( this->partialMatch.has_value()
+       && this->partialNameOrRelPathMatch.has_value() )
+    {
+      throw ParseSearchQueryException(
+        "`partialMatch' and `partialNameOrRelPathMatch' filters "
```

Comment:
Blocker: we decided on single quotes or all user-facing text.

Reply:
Oops I may have replaced every backtick in the file 😉